sdk/core/core-sse/review/core-sse.api.md

```diff
@@ -4,6 +4,14 @@
 
 ```ts
 
+/// <reference types="node" />
+
+// @public
+export function createSseStream(chunkIter: ReadableStream<Uint8Array>): EventMessageStream;
```

Comment:
Nit: not part of this PR I know, but perhaps `chunkStream` would be a better name for the parameter here since it might not be iterable depending on environment?

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/28079/commits/765bc6fc87a2230563ec6b9e0b2c2eeccbbff168